lib/constructor/constructor_lib.lua

```diff
@@ -1932,6 +1918,189 @@ local function map_ini_data_flavor_1(construct_plan, data)
     end
 end
 
+---
+--- INI Mapper Flavor #2
+---
+
+local function map_ini_data_flavor_2(construct_plan, data)
+    debug_log("Found INI flavor 2")
+end
+
+---
+--- INI Mapper Flavor #3
+---
+
+local function map_ini_data_flavor_3(construct_plan, data)
+    debug_log("Found INI flavor 3")
+end
+
+---
+--- INI Mapper Flavor #4
+---
+
+local function map_ini_vehicle_flavor_4(attachment, data, index)
+    local vehicle_main = data["Vehicle"..index]
+
+    constructor_lib.default_vehicle_attributes(attachment)
+    if vehicle_main["Hash"] ~= nil then attachment.hash = vehicle_main["Hash"] end
+    if attachment.model == nil and attachment.hash ~= nil then
+        attachment.model = util.reverse_joaat(attachment.hash)
+    end
+
+    if vehicle_main["Dirt"] ~= nil then attachment.vehicle_attributes.paint.dirt_level = tonumber(vehicle_main["Dirt"]) end
+    if vehicle_main["IsEngineOn"] ~= nil then attachment.vehicle_attributes.options.engine_running = toboolean(vehicle_main["IsEngineOn"]) end
+    if vehicle_main["HeadlightMultiplier"] ~= nil then attachment.vehicle_attributes.headlights.multiplier = tonumber(vehicle_main["HeadlightMultiplier"]) end
+    if vehicle_main["Siren"] ~= nil then attachment.vehicle_attributes.options.siren = toboolean(vehicle_main["Siren"]) end
+
+    if vehicle_main["Doorbroken0"] ~= nil then attachment.vehicle_attributes.doors.broken.frontleft = toboolean(vehicle_main["Doorbroken0"]) end
+    if vehicle_main["Doorbroken1"] ~= nil then attachment.vehicle_attributes.doors.broken.frontright = toboolean(vehicle_main["Doorbroken1"]) end
+    if vehicle_main["Doorbroken2"] ~= nil then attachment.vehicle_attributes.doors.broken.backleft = toboolean(vehicle_main["Doorbroken2"]) end
+    if vehicle_main["Doorbroken3"] ~= nil then attachment.vehicle_attributes.doors.broken.backright = toboolean(vehicle_main["Doorbroken3"]) end
+    if vehicle_main["Doorbroken4"] ~= nil then attachment.vehicle_attributes.doors.broken.hood = toboolean(vehicle_main["Doorbroken4"]) end
+    if vehicle_main["Doorbroken5"] ~= nil then attachment.vehicle_attributes.doors.broken.trunk = toboolean(vehicle_main["Doorbroken5"]) end
+    if vehicle_main["Doorbroken6"] ~= nil then attachment.vehicle_attributes.doors.broken.trunk2 = toboolean(vehicle_main["Doorbroken6"]) end
+
+    if vehicle_main["TyreBurstLF"] ~= nil then attachment.vehicle_attributes.wheels.tires_burst[0] = toboolean(vehicle_main["TyreBurstLF"]) end
```

Comment:
Theres issues with using an integer key here, I think if we use 45 and 47 keys later, Lua will auto-generate keys to fill between and create an issue. To keep this cleaner constructor (and menyoo) often prepend these index keys with a "_" to make them strings. So these should actually be `.tires_burst["_0"]`